fan-xiaomi.js

```diff
@@ -743,7 +848,7 @@ LED
 
         // LED
         let activeElement = fanboxa.querySelector('.c3')
-        if (led_brightness < 2) {
+        if (led_brightness > 0) {
```

Comment:
@ikaruswill This didn't make sense to me. I had intepreted "brightness" to be a percentage, but does `led_brightness` mean something different when coming from attributes via the xiaomi_miio_fan integration?

Reply:
Ugh. Okay, I've updated the PR so that it's only using `> 0` when using the `number.*` entity, and otherwise using `< 2` on the attribute.

To support other models through the `default` platform fully we'd need to also support `select.*` brightness entities, but without an example of that I'm not sure I'd implement it correctly.

If anyone with other fans can share their list of entities and the attributes for each I'd be happy to have a go at broadening support.

Reply:
I have the following entities for my `zhimi.fan.za5` (Pardon the cute name)
<img width="482" alt="image" src="https://user-images.githubusercontent.com/10669647/156597700-19bfd9cb-ddf3-4324-80c0-570570ad6f47.png">

And the following attributes for fan device:
```
speed_list:
  - 'off'
  - low
  - medium
  - high
  - Nature
  - Normal
preset_modes:
  - Nature
  - Normal
direction: null
oscillating: false
speed: Normal
percentage: null
percentage_step: 1
preset_mode: Normal
friendly_name: Gusty
supported_features: 15
```

Reply:
I don't have my za4 connected and accessible right now, but based on what I remember, the LED brightness is set with a select entity, with `bright`, `dim`, `off` options.

It's corroborates with the [HA docs](https://www.home-assistant.io/integrations/xiaomi_miio/):
```
Controls the brightness of the LEDs (bright, dim, off)
```

Reply:
I pushed an update to add a select entity... though I can't really test this. Do you think this is sufficient to push out a beta? I expect there will be some models the LED state doesn't show for, but until we get some more data we can't do much about that.

Reply:
Yeah I think it's all good for us to push out the beta by now. Will do so within the day. Thanks for the help with supporting the select entity for the LED.